apisix/cli/apisix.lua

```diff
@@ -25,7 +25,7 @@ local pkg_path = apisix_home .. "/deps/share/lua/5.1/?.lua;"
 
 -- modify the load path to load our dependencies
 package.cpath = pkg_cpath .. pkg_cpath_org
-package.path  = pkg_path .. pkg_path_org
+package.path  = pkg_path .. pkg_path_org .. ";" .. apisix_home .. "/?.lua"
```

Comment:
It's not a good idea. What if people try to install APISIX to another directory instead of `/usr/local/apisix`, like `/usr/share/lua/5.1`?